### DISCUSSION TITLE
Provide a 'trivy --list-all-tests' switch

### DISCUSSION THREAD

**Discussion by huornlmj** [aquasecurity/trivy#6299] [Ideas]
### Description

Sometimes I need to see the assigned severity of a particular test, but if I cannot see the entire list of Trivy tests as needed then I have to create a source file to scan that has the issue present in it in order to trigger the test and see the result, so that I can see the severity.  Such a switch would be valuable.

### Target

None

### Scanner

Misconfiguration

**Comment by itaysk** [comment]
all checks (and their severities) are documented in https://avd.aquasec.com does that help you?

**Comment by huornlmj** [comment]
Hi.  Not really.  It's too 'mouse-clicky'.  I'm looking for an in-shell solution.  Actually since posting the closest thing switch I found that currently exists was `--include-non-failures              include successes and exceptions, available with '--scanners misconfig'` , but you have to have something to scan for that to work.
